java/pom.xml

```diff
@@ -323,7 +323,7 @@
 
     <distributionManagement>
         <repository>
-            <id>fabric-maven</id>
+            <id>maven-artifactory</id>
```

Comment:
When I generate the settings from our jfrog artifactory, it gives me:

```
<distributionManagement>
    <repository>
        <id>central</id>
        <name>hyperledger-artifactory-primary-0-releases</name>
        <url>https://hyperledger-fabric.jfrog.io/artifactory/fabric-maven</url>
    </repository>
    <snapshotRepository>
        <id>snapshots</id>
        <name>hyperledger-artifactory-primary-0-snapshots</name>
        <url>https://hyperledger-fabric.jfrog.io/artifactory/fabric-maven</url>
    </snapshotRepository>
</distributionManagement>
```

Reply:
The url matches, so that's good. Hopefully the id in the settings.xml that azure generates and the pom will now match and it'll start working again🤞